motion/motionprocessor.go

```diff
@@ -132,7 +140,27 @@ func (mp *MotionProcessor) GetRecentFrame(frame *lepton3.Frame) *lepton3.Frame {
 	return mp.frameLoop.CopyRecent(frame)
 }
 
+// setSunriseSunsetWindow sets the recording window based of todays sunset and sunrise location
+func (mp *MotionProcessor) setSunriseSunsetWindow() {
+	if mp.sunriseSunsetWindow {
+		curTime := time.Now()
+		if mp.nextSunriseCheck.Before(curTime) {
```

Comment:
this might clearer with early returns:
```go
func (mp *MotionProcessor) setSunriseSunsetWindow() {
    if !mp.sunriseSunsetWindow {
        return
    }

    curTime := time.Now()
    if curTime.After(mp.nextSunriseCheck) {
        return
    }

    delay := time.Duration(mp.sunriseSunsetDelay) * time.Minute
    ...
}
```